UIKitDZ/UIKitDZ/MainViewController.swift

```diff
@@ -0,0 +1,169 @@
+        button.layer.borderWidth = 5
+        button.layer.borderColor = #colorLiteral(red: 0.2235294118, green: 0.3568627451, blue: 0.3921568627, alpha: 1)
+        button.addTarget(self, action: #selector(guessNumberAllert), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let resultLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = #colorLiteral(red: 0.9058823529, green: 0.9647058824, blue: 0.9490196078, alpha: 1)
+        label.text = "Результат сложения"
+        label.adjustsFontSizeToFitWidth = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let hiddenNumberLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = #colorLiteral(red: 0.9058823529, green: 0.9647058824, blue: 0.9490196078, alpha: 1)
+        label.text = "Загаданное число"
+
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    let number = "19"
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupViews()
+        setConsrtaints()
+        welcomeAllert()
+        additionAlert()
+    }
+
+    private func setupViews() {
+        view.backgroundColor = #colorLiteral(red: 0.1725490196, green: 0.2, blue: 0.2, alpha: 1)
+
+        view.addSubview(nameLabel)
+        view.addSubview(additionButton)
+        view.addSubview(guesTheNumberButton)
+        view.addSubview(resultLabel)
+        view.addSubview(hiddenNumberLabel)
+    }
+
+    func welcomeAllert() {
+        let welcomeAllert = UIAlertController(title: "Привет",
+                                              message: "Введите свое имя",
+                                              preferredStyle: .alert)
+        let action = UIAlertAction(title: "OK", style: .default) { _ in
+            guard let text = welcomeAllert.textFields?.first else { return }
+            self.nameLabel.text = text.text
+        }
+        welcomeAllert.addTextField { _ in
+        }
+        welcomeAllert.addAction(action)
+        self.present(welcomeAllert, animated: true, completion: nil)
+    }
+
+    @objc func additionAlert() {
+        let additionAllert = UIAlertController(title: "Сложение",
+                                               message: "Введите числа",
+                                               preferredStyle: .alert)
+        let additionAction = UIAlertAction(title: "узнайте сумму двух чисел", style: .default) { _ in
+            guard let numberOne = additionAllert.textFields?.first else { return }
+            guard let numberTwo = additionAllert.textFields?.last else { return }
+            let intNumberOne = Int(numberOne.text ?? "0") ?? 0
+            let intNumberTwo = Int(numberTwo.text ?? "0") ?? 0
+            self.resultLabel.text = String(intNumberOne + intNumberTwo)
+        }
+        additionAllert.addTextField { _ in}
```

Comment:
```suggestion
        additionAllert.addTextField()
        additionAllert.addTextField()
```
 не нужны пустые кложеры